Classes/Twig/Extension/DBRelationExtension.php

```diff
@@ -70,8 +70,12 @@ public function lookupRelation(EnvironmentTwig $env, \Tx_Rnbase_Domain_Model_Bas
         $alias = $env->getConfigurations()->get($confId . 'join.alias');
         $field = $env->getConfigurations()->get($confId . 'join.field');
         if (! $alias || ! $field) {
-            throw new \Exception('Verify configuration for relation \'' . htmlspecialchars($arguments['relation']) . '\'.
-                            Table alias or field not found. Full typoscript path: ' . $confId);
+            throw new \Exception(sprintf(
+                "Verify config for relation '%s' Table alias or field not found. Full typoscript path: %s",
+                    htmlspecialchars($arguments['relation']),
+                    $confId
+                )
+            );
```

Comment:
There are still errors in lines [73-77](https://rc.dmknet.de/group/Team-4?msg=xTzWvviauk66rdv7j).

Reply:
Keep calm and force push :P